awsconfigs/apps/pipeline/s3/deployment_patch.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kubeflow-pipelines-profile-controller
+spec:
+  replicas: 1
+  template:
+    spec:
+      containers:
+      - name: profile-controller
+        env:
+        - name: MINIO_SERVICE_REGION
+          valueFrom:
+            configMapKeyRef:
+              name: pipeline-install-config
+              key: minioServiceRegion
```

Comment:
where are we modifying this configMap to have the region ?

Reply:
rds-s3 configuration is doing that and it is getting deployed as a part of this file https://github.com/awslabs/kubeflow-manifests/blob/main/awsconfigs/apps/pipeline/kustomization.yaml#L10